src/ui/timers_main_area.ui

```diff
@@ -80,55 +110,63 @@
        <height>61</height>
       </size>
      </property>
-     <layout class="QHBoxLayout" name="horizontalLayout">
-      <property name="spacing">
-       <number>0</number>
-      </property>
+     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,0,0,0,1">
       <property name="leftMargin">
-       <number>0</number>
+       <number>3</number>
       </property>
       <property name="topMargin">
-       <number>0</number>
+       <number>3</number>
       </property>
       <property name="rightMargin">
-       <number>0</number>
+       <number>3</number>
       </property>
       <property name="bottomMargin">
-       <number>0</number>
+       <number>3</number>
       </property>
-      <item>
-       <widget class="QLabel" name="label">
-        <property name="sizePolicy">
-         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-          <horstretch>0</horstretch>
-          <verstretch>0</verstretch>
-         </sizepolicy>
+      <property name="spacing">
+       <number>3</number>
+      </property>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_hours">
+        <property name="text">
+         <string>Hours:</string>
         </property>
-        <property name="minimumSize">
```

Comment:
Will check that in the timers_main_area.ui design.  That one is `fixed` I think but this suggests it needs checking.  (By comparison, I like the fact that I am not longer limited to only showing a fixed number of trigger items as I can drag the separator to show more or less as needed - it is just a case of ensuring realistic minimums. :grinning: )